src/Twig/SecurityPolicyContainerAware.php

```diff
@@ -140,16 +140,16 @@ public function checkMethodAllowed($obj, $method): void
 
     /**
      * @param object $obj
-     * @param string $property
```

Comment:
I opened https://github.com/twigphp/Twig/pull/3664/files ^^

Reply:
👍 Since it requires either changing the property name or adding a suppress we will need to come back here once it is merged an released, so I prefer leave it as is and fix it again later.